TcUnit-Runner/XmlUtilities.cs

```diff
@@ -61,5 +61,20 @@ public static string SetDisabledAndAndAutoStartOfRealTimeTaskXml(string rtXml, b
             }
         }
 
+        public static bool GetPinnedVersionXML(string TwinCATProjectFilePath)
```

Comment:
I don't get this... the method is called GetPinnedVersionXML, but it returns a boolean?...
IsTwinCATProjectPinned()?

Reply:
I tried to keep the same spelling as for the other XmlUtilities.
e.g. 
`public static string GetItemNameFromRealTimeTaskXML(string rtXml)`

But yes, the method returns a boolean that tells you if the version is pinned or not. For this it does not use the `ITcSysManager13.TcVersionFixed` but adapts the information from the XML.

like the other xml methods

Reply:
Sounds good. I think all is fine if you just rename the method.